backend/resolvers.js

```diff
@@ -0,0 +1,72 @@
+const { neo4jgraphql } =  require ('neo4j-graphql-js');
+
+const jwt = require('jsonwebtoken');
+const { AuthenticationError } = require('apollo-server-errors');
+const decryptedToken = require('./jwtDecoder');
+
+const todos = [
+    {
+      title: 'SDF - Task 3',
+    },
+    {
+      title: 'SDF - Task 2',
+    },
+  ];
+  const users = [
+    {
+      username: "user1",
+      password: "12345"
+    },
+    {
+      username: "user2",
+      password: "54321"
+    }
+  ];
+
+const resolvers = {
+    Query: {
+      todos: (object, params, ctx, resolveInfo) => {
+        const decrypted = decryptedToken(ctx.token);
```

Comment:
Remove unused local variable `decrypted`. All you want to do is to throw an error. So:
```suggestion
        decryptedToken(ctx.token);
```
oh and you should avoid past tense for a method name:
```suggestion
        decryptToken(ctx.token);
```